src/components/Tabs/components/Tab/Tab.tsx

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from "react"
+import { Tab as TabType } from "../../types"
+
+export interface TabProps {
+  children: ReactNode
+  label: string
+  id: TabType["id"]
+  disabled?: boolean
+}
+
+export const Tab = ({ children }: TabProps) => <>{children}</>
```

Comment:
>I believe the Tab and TabPanel components are essentially the same component

Ha ha, you're completely right! I'll need some time to look through your solution, but if both require some magic to prevent `selected` from leaking, the question then is, "which requires less magic?"

There's definitely a part of me that wants to go back and refactor this whole thing now that I have a better sense of what's required—taking into account what we've learned. BUT, I think I'll merge this for now and open a new PR that addresses those changes specifically to make reviewing all that a bit more manageable.